src/masternode-payments.cpp

```diff
@@ -785,6 +785,40 @@ void CMasternodePayments::ProcessBlock(int nBlockHeight)
     nLastBlockHeight = nBlockHeight;
 }
 
+bool CMasternodePayments::CreateMNWinnerPayment(const CTxIn& mnVinWinner, int paymentBlockHeight, const CScript& payeeScript)
```

Comment:
This is currently only used in the tests.
Maybe the intention was to call it also from within `CMasternodePayments::ProcessBlock` (thus adding extra checks before the relay)?
If, instead, the intention is to use it only for the tests, then better to implement this directly inside mnpayments_tests.cpp